go.mod

```diff
@@ -0,0 +1,45 @@
+module github.com/hairyhenderson/gomplate
+
+go 1.13
+
+require (
+	github.com/Masterminds/goutils v1.1.0
+	github.com/Shopify/ejson v1.2.1
+	github.com/armon/go-metrics v0.0.0-20190430140413-ec5e00d3c878 // indirect
+	github.com/aws/aws-sdk-go v1.25.5
+	github.com/boltdb/bolt v1.3.1
+	github.com/docker/libkv v0.2.1
+	github.com/dustin/gojson v0.0.0-20160307161227-2e71ec9dd5ad // indirect
+	github.com/frankban/quicktest v1.5.0 // indirect
+	github.com/fullsailor/pkcs7 v0.0.0-20190404230743-d7302db945fa
+	github.com/google/uuid v1.1.1
+	github.com/gosimple/slug v1.7.0
+	github.com/hairyhenderson/toml v0.3.1-0.20191004034452-2a4f3b6160f2
+	github.com/hashicorp/consul/api v1.2.0
+	github.com/hashicorp/go-immutable-radix v1.1.0 // indirect
+	github.com/hashicorp/go-retryablehttp v0.6.2 // indirect
+	github.com/hashicorp/go-sockaddr v1.0.2
+	github.com/hashicorp/golang-lru v0.5.3 // indirect
+	github.com/hashicorp/serf v0.8.5 // indirect
+	github.com/hashicorp/vault/api v1.0.4
+	github.com/joho/godotenv v1.3.0
+	github.com/pierrec/lz4 v2.3.0+incompatible // indirect
+	github.com/pkg/errors v0.8.1
+	github.com/rainycape/unidecode v0.0.0-20150907023854-cb7f23ec59be // indirect
+	github.com/smartystreets/goconvey v0.0.0-20190731233626-505e41936337 // indirect
+	github.com/spf13/afero v1.2.2
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/stretchr/testify v1.3.0
+	github.com/ugorji/go/codec v1.1.7
+	github.com/zealic/xignore v0.3.3
+	golang.org/x/crypto v0.0.0-20191002192127-34f69633bfdc
+	golang.org/x/net v0.0.0-20191003171128-d98b1b443823 // indirect
+	golang.org/x/sys v0.0.0-20191003212358-c178f38b412c // indirect
+	golang.org/x/text v0.3.2 // indirect
+	golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0 // indirect
+	gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405
+	gopkg.in/yaml.v3 v3.0.0-20190924164351-c8b7dadae555
+	gotest.tools/v3 v3.0.0
+	k8s.io/client-go v11.0.0+incompatible
```

Comment:
Why do you pull in k8s.io's client?

Reply:
that's a good question 😲... I'll dig into that tomorrow morning 🛌

Reply:
oh actually, that's a simple answer - it's here: https://github.com/hairyhenderson/gomplate/blob/7724a8d8e6b0437b151cbfc5f4ca2ee5695d0bf8/coll/jsonpath.go#L7

Reply:
I see. I think this is a very big dependency for such a small package. I wonder if you could look into this to make this package more leaner. Thanks already for moving to Go modules and I appreciate. I think the less dependencies the better. You or any other user, they shouldn't pull down a big dependency like this.

Reply:
@fatih maybe I'm misunderstanding how go modules work, but is all of `k8s.io/client-go` downloaded just to get that one package? When I `go mod vendor` it, only that package is included:

```
$ du -h vendor/k8s.io/client-go
40K     vendor/k8s.io/client-go/util/jsonpath
40K     vendor/k8s.io/client-go/util
20K     vendor/k8s.io/client-go/third_party/forked/golang/template
20K     vendor/k8s.io/client-go/third_party/forked/golang
20K     vendor/k8s.io/client-go/third_party/forked
20K     vendor/k8s.io/client-go/third_party
72K     vendor/k8s.io/client-go
```

I am seeing that when I `go get` it, it pulls down all of `k8s.io/client-go` into my module cache, which is ~6MB and 1000+ files 🤔...